Procfile

```diff
@@ -1,3 +1,4 @@
+setup: cd playbook-website && node scripts/global-props.mjs
```

Comment:
This will run when the server starts, adding negligible time while providing the names of all global props in a module.